webapp/portlet/src/main/webapp/vue-apps/attach-image/components/PreviewAttachment.vue

```diff
@@ -0,0 +1,144 @@
+<!--
```

Comment:
Please rename File to `AttachmentPreviewDialog.vue` and component to `attachment-preview-dialog`

Reply:
done